relationships/synthesis/APM-APPLICATION-to-INFRA-REDSHIFTCLUSTER.yml

```diff
@@ -0,0 +1,22 @@
+relationships:
+  - name: apmApplicationCallsInfraRedshiftCluster
+    version: "1"
+    origins:
+      - Distributed Tracing
+    conditions:
+      - attribute: eventType
+        anyOf: [ "Span" ]
+      - attribute: configuration.endpoint.address
```

Comment:
was this agreed upon with the agents? I thought this was going to `server.address` as that's the span attribute we assign to the connection host for database spans